theme_anelusia/views/snippets/s_product_list.xml

```diff
@@ -36,7 +36,7 @@
     </xpath>
     <!-- Item #4 - Img -->
     <xpath expr="(//img)[4]" position="attributes">
-        <attribute name="src">/web_editor/image_shape/website.s_product_list_default_image_4/web_editor/geometric/geo_square_1g</attribute>
```

Comment:
Is not this line a pure unrelated bug fix? If that's the case, you should target the right stable version in another PR with a dedicated commit :+1:

Reply:
Hey @qsm-odoo 👋 Indeed ! I created a new PR for that, thanks !
https://github.com/odoo/design-themes/pull/689